### PR TITLE
Added Compiler and Runtime Args to io bot

### PR DESCRIPTION
**Pull request by isFakeAccount**
- Added Compiler and Runtime Args to the io bot. This can be accessed from the `Set Inputs` button, previously called `Set STDIN`. 
- Made some minor modifications to the output of the code. The goal was to make it easy to copy someone's Compiler/Runtime args or STDIN. With the `IN: trag1c`, you'd have to manually erase the `IN: ` part.
image
- There is now a limit on the numbers of chars you enter. Following are the limits
- STDIN: 1024 chars
- COMPTIME ARGS: 255 chars
- RUNTIME ARGS: 255 chars

### CODE REVIEW

**Comment by isFakeAccount** [comment]
Issue Link: #7

File: bot/plugins/instance.py [hunk 1]
```diff
@@ -30,12 +30,15 @@ class ComponentID(enum.StrEnum):
...
-    STDIN = "stdin"
+    INPUTS = "inputs"
 
 
 class ModalID(enum.StrEnum):
     LANGUAGE = "language"
+    INPUTS = "inputs"
     STDIN = "stdin"
+    COMPTIME = "comptime"
+    RUNTIME = "runtime"
```
Comment (circuitsacul) [inline bot/plugins/instance.py]: These are unused I think?

File: bot/plugins/instance.py [hunk 2]
```diff
@@ -210,6 +209,46 @@ async def on_component_interaction(event: hikari.InteractionCreateEvent) -> None
...
+
+    runtime_modal_row = event.app.rest.build_modal_action_row()
+    runtime_modal_row.add_text_input(
+        ModalID.RUNTIME,
+        "Runtime Args",
+        value=msg_instance.runtime_args or hikari.UNDEFINED,
+        required=False,
+        style=hikari.TextInputStyle.PARAGRAPH,
+        max_length=255,
+        placeholder="Command Line Parameters - 1 per line",
```
Comment (circuitsacul) [inline bot/plugins/instance.py]: Let's use a different placeholder here than for compile args

File: bot/plugins/instance.py [hunk 3]
```diff
@@ -210,6 +209,46 @@ async def on_component_interaction(event: hikari.InteractionCreateEvent) -> None
     await inst.execute()
 
 
+def create_input_args_modal(
+    event: hikari.InteractionCreateEvent,
+    msg_instance: Instance,
+):
```
Comment (circuitsacul) [inline bot/plugins/instance.py]: This needs a return type. `list[...]`. Whatever type `build_modal_action_row` returns

File: bot/plugins/instance.py [hunk 4]
```diff
@@ -488,11 +529,20 @@ async def execute(self) -> None:
             out.append("No runtime selected.")
 
         stdin = self.stdin or ""
-        formatted_stdin = "\n".join(
-            f"\x1b[1;33mIN:\x1b[0m {line}" for line in stdin.splitlines()
+        formatted_stdin = "\n".join(f"{line}" for line in stdin.splitlines())
```
Comment (circuitsacul) [inline bot/plugins/instance.py]: `"\n".join("f{line}" for line in stdin.splitlines())` is the same as just using `stdin`. This was just here to add the formatting before it.
```suggestion
```

**Review by circuitsacul — CHANGES_REQUESTED** [review]

File: bot/plugins/instance.py (hunk 1 above)
Comment (isFakeAccount) [reply bot/plugins/instance.py]: They are.

File: bot/plugins/instance.py (hunk 2 above)
Comment (isFakeAccount) [reply bot/plugins/instance.py]: Any suggestions?

Comment (circuitsacul) [reply bot/plugins/instance.py]: Runtime Args - 1 per line

File: bot/plugins/instance.py (hunk 1 above)
Comment (isFakeAccount) [reply bot/plugins/instance.py]: image

I am using them here.

Comment (isFakeAccount) [reply bot/plugins/instance.py]: Line 230

File: bot/plugins/instance.py (hunk 4 above)
Comment (isFakeAccount) [reply bot/plugins/instance.py]: Good catch. I removed the `formatted_comptime_args` and the `formatted_runtime_args` as well.

File: bot/plugins/instance.py (hunk 3 above)
Comment (isFakeAccount) [reply bot/plugins/instance.py]: Fixed

File: bot/plugins/instance.py (hunk 1 above)
Comment (isFakeAccount) [reply bot/plugins/instance.py]: Can you clarify this one?

**Review by circuitsacul — APPROVED** [review]

**Pull request merged**